responsive-style.css

```diff
@@ -0,0 +1,156 @@
+@media screen and (min-width: 768px) {
+  /*  .welcome {
+    padding: auto 240PX;
+  }
+*/
```

Comment:
- [ ] Please remove this commented line of code, it is not a good practice as it will make your code looks unclean and not professional